src/usr/local/bin/install-buildpack

```diff
@@ -28,7 +28,13 @@ fi
 
 # env helper, loads tool specific env
 cat >> $BASH_ENV <<- EOM
-export BUILDPACK=1 USER_NAME="${USER_NAME}" USER_ID="${USER_ID}"
+export BUILDPACK=1 USER_NAME="${USER_NAME}" USER_ID="${USER_ID}" USER_HOME="/home/${USER_NAME}"
+
+# openshift override unknown user home
+if [ "\${EUID}" != 0 ] && [ "\${HOME}" == "/" ]; then
```

Comment:
Home is `/` if run as a random ID?

Reply:
yes, at least on docker. but removed that condition, so only root is checked